tools/c7n_tencentcloud/c7n_tencentcloud/query.py

```diff
@@ -279,7 +287,7 @@ def get_resource_query_params(self):
     def resources(self):
         params = self.get_resource_query_params()
         resources = self.source.resources(params)
-
+        resources = self.augment(resources)
```

Comment:
augment should ideally defer to the source as its typically source specific.

Reply:
How about we do a refactor to make these changes of augment/qcs/... in a special PR but not on this resource level. We can change each resource that has been implemented to follow the new pattern after that.

Reply:
sounds good